### PR TITLE
Create feature load()

### PR DESCRIPTION
**Pull request by tyuyang**
Also created createNewFile(), Storage() and relevant UI functions

### CODE REVIEW

File: src/main/java/seedu/bankwithus/Ui.java [hunk 1]
```diff
@@ -3,6 +3,14 @@
 public class Ui {
     
     public void showLoadingError() {
```
Comment (Sherlock-YH) [inline src/main/java/seedu/bankwithus/Ui.java:5]: Maybe try to rename it like creatingNewFile to comply with the message show to the user

**Comment by xiaoge26** [comment]
I think it looks good! It can be merged after resolving the IO redirection test failures.

File: src/main/java/seedu/bankwithus/Ui.java (hunk 1 above)
Comment (tyuyang) [reply src/main/java/seedu/bankwithus/Ui.java:5]: Done

**Comment by tyuyang** [comment]
Changed EXPECTED.TXT, BWU.java, Ui.java, and build.gradle. Should not have any issues with merging as of 0920 12 03 2023.

**Review by xiaoge26 — APPROVED** [review]
Ready to merge

**Pull request merged**